env.ts

```diff
@@ -6,8 +6,8 @@ const filepath = ".env";
 // Check to see if ngrok is up and running
 (async function main() {
 
-  // Does .env exists?
-  if (!fs.existsSync(filepath)) {
+  // Does .env exist and is the app running locally?
+  if (!fs.existsSync(filepath) && !process.env.MICROS_ENV) {
```

Comment:
I'm not sure I understand the point of this?

Reply:
There's no env file in micros so the deployment fails.

Reply:
yeah, but micros doesn't call `npm start` :)

Reply:
Oh I think it is because of

`ENV NODE_ENV development` in Dockerfile.prod

It was failing for me as well. I am fixing it in my PR: https://github.com/integrations/jira/pull/503/files

Reply:
ah right, sorry.  Just change it to `production` and it'll work

Reply:
and of course, remove the added conditional in env.ts